docs/guide/java_object_graph_guide.md

```diff
@@ -93,6 +93,32 @@ public class Example {
 }
 ```
 
+FuryBuilder Settings:
+
+| Parameter Name                    | Default Value                                       | Considerations                                                                                                                                              |
+| ---------------------------------- | --------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `language`                        | `Language.JAVA`                                     | If using Fury exclusively for Java, setting it to `Language#JAVA` can improve performance.                                                                    |
+| `trackingRef`                     | `false`                                             | Enable when there are circular references. Otherwise, it may cause a `StackOverflowError` during serialization.                                             |
+| `basicTypesRefIgnored`            | `true`                                              | Influence on `Serializer#needToWriteRef` for basic types.                                                                                                   |
```

Comment:
this is barely used by users, couldwe remove this option? The more we provide,the more users need to understand

Reply:
Done.